_includes/meta.html

```diff
@@ -0,0 +1,11 @@
+{% if page.title %}
+<meta property="og:title" content="{{ page.title }}" />
+{% endif %} 
+
+{% if page.description %}
+<meta property="og:description" content="{{ page.description }}" />
+{% endif %} 
+
+{% if page.cover-photo %}
+<meta property="og:image" content="{{ page.cover - photo }}" />
```

Comment:
ups, sorry there is a typo here. It should be `page.cover-photo`

Reply:
@sauloperez  Fixed the typo. Thanks for pointing out